static/js/util/util.js

```diff
@@ -191,6 +193,42 @@ export function validateProfile(profile, requiredFields, messages) {
 }
 
 /* eslint-enable camelcase */
+/*
+check that the profile is complete. we make the assumption that a
+complete profile consists of:
+  - a valid personal tab
+  - an entry for 'currently employed', and a work history entry if
+    `currently employed == 'yes'`
+*/
+export function profileComplete(profile) {
```

Comment:
Can you rename this `validateProfileComplete`? Can you also return `false` instead of `true`, since the object returned is truthy when there are validation errors?